openhtf/util/measurements.py

```diff
@@ -143,6 +143,11 @@ def __setattr__(self, attr, value):
     if attr == 'dimensions':
       self._initialize_value()
 
+  def __setstate__(self, state):
```

Comment:
can you add a comment as to why this is here?  i think it's the cleanest way to do what you need, but still isn't super obvious why without a comment.